pkg/recordset/recordset.go

```diff
@@ -178,6 +213,42 @@ func getStacks(cl client.StackDescribeLister, re *regexp.Regexp, installation st
 	return result, nil
 }
 
+func filterStacksByStatus(input []cloudformation.Stack, statuses []string) []cloudformation.Stack {
+	output := []cloudformation.Stack{}
+
+	for _, stack := range input {
+		if stackHasStatus(stack, statuses) {
+			output = append(output, stack)
+		}
+	}
+
+	return output
+}
+
+func excludeStacksByStatus(input []cloudformation.Stack, statuses []string) []cloudformation.Stack {
```

Comment:
In functional terms again (sorry I've been there for too long :D) this should be called `dropStacksByStatus`.

Reply:
updating this.

haven't been there much as you can see